io.catenax.pcf/6.0.0/Pcf.ttl

```diff
@@ -0,0 +1,841 @@
+   samm:characteristic :PositiveEmissionsTrait ;
+   samm:exampleValue "2.5"^^xsd:decimal .
+
+:fossilCarbonContent a samm:Property ;
+   samm:preferredName "Fossil Carbon Content"@en ;
+   samm:description "Optional: Fossil carbon amount embodied in a product as specified in the technical specifications for PCF Data Exchange (Version 2.1.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. Must be calculated with kgC (kilogram Carbon) / declaredUnit equal to or greater zero; WBCSD specific extension, in Catena-X specified by a calculated value."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20231207/> ;
+   samm:characteristic :PositiveEmissionsTrait ;
+   samm:exampleValue "0.1"^^xsd:decimal .
+
+:biogenicCarbonContent a samm:Property ;
+   samm:preferredName "Biogenic Carbon Content"@en ;
+   samm:description "Mandatory starting 2025: Biogenic carbon amount embodied in a product as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.1.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. Must be calculated with kgC (kilogram Carbon) / declaredUnit equal to or greater zero."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20231207/> ;
+   samm:characteristic :PositiveEmissionsTrait ;
+   samm:exampleValue "0.0"^^xsd:decimal .
+
+:distributionStageAircraftGhgEmissions a samm:Property ;
+   samm:preferredName "Distribution Stage Aircraft GHG Emissions"@en ;
+   samm:description "Optional: GHG (Greenhouse Gas) emissions for the distribution stage resulting from aircraft engine usage for the transport of the product as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :PositiveEmissionsTrait ;
+   samm:exampleValue "0.0"^^xsd:decimal .
+
+:DeclaredUnitCharacteristic a samm-c:Enumeration ;
+   samm:preferredName "Declared Unit Characteristic"@en ;
+   samm:description "Unit of analysis of the product with accepted values as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. For countable products/ components/ materials, Catena-X for example adds the unit \"piece\" to the value list specified by WBCSD."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "liter" "kilogram" "cubic meter" "kilowatt hour" "megajoule" "ton kilometer" "square meter" "piece" ) .
+
+:StrictlyPositiveDecimalTrait a samm-c:Trait ;
```

Comment:
please add preferredName and description to all traits

Reply:
Missing preferred names and descriptions added, see commit.